libs/s25main/RoadSegment.cpp

```diff
@@ -109,13 +109,13 @@ void RoadSegment::Serialize(SerializedGameData& sgd) const
  */
 void RoadSegment::SplitRoad(noFlag* splitflag)
 {
-    // Flagge 1 _________ Diese Flagge _________ Flagge 2
-    //         |       unterbrochener Weg       |
+    // Flag 1 _________ This flag _________ Flag 2
+    //          | broken path |
```

Comment:
This is kind of ASCII-art for visualization, so keep the number of characters:
```suggestion
    // Flag 1 _________ This flag _________ Flag 2
    //         |          broken road           |
```